src/main/java/ch/njol/skript/lang/function/Signature.java

```diff
@@ -63,16 +63,26 @@
 	 * References (function calls) to function with this signature.
 	 */
 	final Collection<FunctionReference<?>> calls;
-	
-	public Signature(String script, String name, Parameter<?>[] parameters, @Nullable final ClassInfo<T> returnType, boolean single) {
+
+	/**
+	 * References (function calls) to function with this signature.
+	 */
+	final String originClassPath;
```

Comment:
Created to get the addon which registered this function (same as other elements) but not yet used as stated [here](
https://github.com/SkriptLang/Skript/pull/4319/files/ee19cb92e27d958ab7b1f0376194ed5ddcfc37d6?short_path=0b5ca11#diff-6762fbc1d43ae0e614d3d4bde26a31d26ddfb076dab557078c1f66f9dc14a80eR471-R474) and [here](https://github.com/SkriptLang/Skript/pull/4319/files/ee19cb92e27d958ab7b1f0376194ed5ddcfc37d6?short_path=0b5ca11#diff-6762fbc1d43ae0e614d3d4bde26a31d26ddfb076dab557078c1f66f9dc14a80eR708-R711)

![image](https://user-images.githubusercontent.com/20037329/155845196-ec2e9d4a-274f-456e-b86f-595706edccdf.png)
![image](https://user-images.githubusercontent.com/20037329/155845338-0649766b-2186-4e61-8cd8-3e940b3d6e4d.png)

(in case links don't load)

However, its javadoc is wrong

Reply:
Unresolved until I change its javadoc